core/src/main/java/com/radixdlt/sync/LocalSyncService.java

```diff
@@ -430,10 +430,20 @@ private SyncState processSyncResponse(
     log.trace("LocalSync: Received sync response from {}", sender);
 
     if (!currentState.waitingForResponseFrom(sender)) {
-      log.warn("LocalSync: Received unexpected sync response from {}", sender);
+      log.trace("LocalSync: Received unexpected sync response from {}", sender);
+      metrics.sync().unexpectedSyncResponses().inc();
       return currentState;
     }
 
+    final var responseTailStateVersion =
+        syncResponse.getLedgerExtension().getTail().getLedgerHeader().getStateVersion();
+    final var currentStateVersion = currentState.getCurrentHeader().getStateVersion();
+    if (responseTailStateVersion <= currentStateVersion) {
+      log.trace("LocalSync: Received a stale sync response from {}", sender);
+      metrics.sync().staleSyncResponses().inc();
```

Comment:
_(if you have time for such nitpicking)_ If the metric had a `sender` label, then the log would not be needed at all.

Reply:
I would keep the log either way.